test/e2e/scenario/eventBridge.scenario.ts

```diff
@@ -101,4 +101,34 @@ describe('bridge present', () => {
       expect(intakeRegistry.replayRecords.length).toBeGreaterThan(0)
       expect(intakeRegistry.hasOnlyBridgeRequests).toBe(true)
     })
+
+  createTest('should only send records when the recording is started')
+    .withRum({ startSessionReplayRecordingManually: true })
+    .withEventBridge()
+    .run(async ({ intakeRegistry }) => {
+      expect(intakeRegistry.replayRecords.length).toEqual(0)
+
+      await browserExecute(() => {
+        window.DD_RUM!.startSessionReplayRecording()
+      })
+
+      // wait for recorder to be properly started
+      await browser.pause(200)
+
+      expect(intakeRegistry.replayRecords.length).toBeGreaterThan(0)
+      expect(intakeRegistry.hasOnlyBridgeRequests).toBe(true)
+
+      const preStopRecordsCount = intakeRegistry.replayRecords.length
+      await browserExecute(() => {
+        window.DD_RUM!.stopSessionReplayRecording()
+
+        // trigger a new record
+        document.body.appendChild(document.createElement('li'))
+      })
+
+      await flushEvents()
+
+      const postStopRecordsCount = intakeRegistry.replayRecords.length - preStopRecordsCount
+      expect(postStopRecordsCount).toEqual(0)
```

Comment:
**❓ question:** ‏I'm not sure to see why we need all this in this test? Isn't something like this enough?

```ts
  createTest('should only send records when the recording is started')
    .withRum()
    .withEventBridge()
    .run(async ({ intakeRegistry }) => {
      await flushEvents()

      expect(intakeRegistry.replayRecords.length).toBeGreaterThan(0)
      expect(intakeRegistry.hasOnlyBridgeRequests).toBe(true)
```

Reply:
The test also test that no records are sent through the bridge if session replay is stopped. I used startSessionReplayRecordingManually: true to be more explicite on what is the code doing. But I could simplify like so I guess
```suggestion
    .withRum()
    .withEventBridge()
    .run(async ({ intakeRegistry }) => {
      // wait for recorder to be properly started
      await browser.pause(200)

      expect(intakeRegistry.replayRecords.length).toBeGreaterThan(0)
      expect(intakeRegistry.hasOnlyBridgeRequests).toBe(true)

      const preStopRecordsCount = intakeRegistry.replayRecords.length
      await browserExecute(() => {
        window.DD_RUM!.stopSessionReplayRecording()

        // trigger a new record
        document.body.appendChild(document.createElement('li'))
      })

      await flushEvents()

      const postStopRecordsCount = intakeRegistry.replayRecords.length - preStopRecordsCount
      expect(postStopRecordsCount).toEqual(0)
```
Wdyt?

Reply:
Mmh I don't know if it's truly necessary, as this behavior is unrelated to the event bridge?

If you think it is, you could add another test case so it's more explicit

Reply:
Since on `stopSessionReplayRecording`  the segmentCollection is stopped and not the bridge I prefer to check it. I'll make a separated test